template-xml/domain/src/main/java/co/nimblehq/template/xml/domain/exceptions/Exceptions.kt

```diff
@@ -1,6 +1,6 @@
-package co.nimblehq.template.domain.exceptions
```

Comment:
same for `domain` package 🤔

Reply:
@luongvo We still have `domain` here ☝️ `co.nimblehq.template.xml.domain`